src/components/ChartView/ChartView.tsx

```diff
@@ -65,7 +72,12 @@ class ChartView extends React.Component<IChartViewProps> {
                 <aside className="ChartViewSidebar bg-light margin-v-big padding-h-normal padding-b-normal">
                   <div className="ChartViewSidebar__section">
                     <h2>Usage</h2>
-                    <ChartDeployButton push={push} version={version} deployChart={deployChart} />
+                    <ChartDeployButton
+                      push={push}
+                      version={version}
+                      deployChart={deployChart}
+                      values={values || ""}
```

Comment:
We should use the `defaultProps` static method in the ChartDeployButton class to set the default values https://stackoverflow.com/questions/37282159/default-property-value-in-react-component-using-typescript

Reply:
Are you saying in place of `values={values || ""}`? `values={values || ""}` is necessary because during initial render of ChartView `values` might be `null`. If we made `defaultProps = {values: ''}` for `ChartDeployButton`, we would still need to do an if/else for which ChartDeployButton to render in ChartView (with or without `values` prop)

Reply:
I guess I was thinking that the Interface could make define values as `values?: string` and then the default would be set by defaultProps, but I'm not sure what the best way is really (and whether in that approach would still require the `|| ""`.

If you think the `values={values || ""}` then I'm good with that, it was just a thought.